hack/virtc/main.go

```diff
@@ -227,7 +228,14 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 		Memory: vmMemory,
 	}
 
+	id := context.String("pod-id")
```

Comment:
Yes, but he also means that virtcontainers should be responsible for this, and we could stop to export ID in PodConfig.
type PodConfig struct {
id
....
}